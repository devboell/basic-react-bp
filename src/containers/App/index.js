import React from 'react'

import reactjs2Img from './reactjs2.jpg'

const App = () =>
  <div>
    <img src="images/reactjs.jpg" alt="" />
    <div>React</div>
    <img src={reactjs2Img} alt="" />
  </div>

export default App
