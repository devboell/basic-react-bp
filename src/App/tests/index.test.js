import React from 'react'

import App from '../index'

test('App component', () => {
  const wrapper = mount(<App />)
  expect(wrapper.find('div').at(1).text()).toBe('React')
  expect(wrapper).toMatchSnapshot()
})
