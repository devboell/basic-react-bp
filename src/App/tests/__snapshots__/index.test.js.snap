// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component 1`] = `
<App>
  <div>
    <img
      alt=""
      src="images/reactjs.jpg"
    />
    <div>
      React
    </div>
    <img
      alt=""
      src="test-file-stub"
    />
  </div>
</App>
`;
